website/pages/en/network/delegating.mdx

```diff
@@ -71,28 +73,32 @@ Another thing a Delegator has to consider is what proportion of the Delegation P
 
 ![Share formula](/img/Share-Forumla.png)
 
-Using this formula, we can see that it is actually possible for an Indexer who is offering only 20% to Delegators, to actually be giving Delegators an even better reward than an Indexer who is giving 90% to Delegators.
+Using this formula, we can see that it is possible for an Indexer offering only 20% to Delegators to actually provide a better reward than an Indexer giving 90%.
 
-A Delegator can therefore do the math to determine that the Indexer offering 20% to Delegators, is offering a better return.
+Therefore, a Delegator can do the math to determine that the Indexer offering 20% to Delegators, is offering a better return.
 
 ### Considering the delegation capacity
 
 Another thing to consider is the delegation capacity. Currently, the Delegation Ratio is set to 16. This means that if an Indexer has staked 1,000,000 GRT, their Delegation Capacity is 16,000,000 GRT of Delegated tokens that they can use in the protocol. Any delegated tokens over this amount will dilute all the Delegator rewards.
 
-Imagine an Indexer has 100,000,000 GRT delegated to them, and their capacity is only 16,000,000 GRT. This means effectively, 84,000,000 GRT tokens are not being used to earn tokens. And all the Delegators, and the Indexer, are earning way less rewards than they could be.
+Imagine an Indexer with 100,000,000 GRT delegated to them, but their capacity is only 16,000,000 GRT. This means effectively, 84,000,000 GRT tokens are not being used to earn tokens. So, all the Delegators and the Indexer, are earning way less rewards than they could be.
 
-Therefore a Delegator should always consider the Delegation Capacity of an Indexer, and factor it into their decision making.
+Therefore, a Delegator should always consider the Delegation Capacity of an Indexer, and factor it into their decision making.
 
 ## Delegator FAQs and Bugs
 
 ### MetaMask "Pending Transaction" Bug
 
-**When I try to delegate my transaction in MetaMask appears as "Pending" or "Queued" for longer than expected. What should I do?**
+**When I try to delegate my transaction in MetaMask, it appears as "Pending" or "Queued" for longer than expected. What should I do?**
```

Comment:
This should probably be a heading (`####`). But since this is the only question in the FAQ section, maybe the whole section should be re-structured/re-thought? Or do we plan on adding more?

Reply:
```suggestion
### When I try to delegate my transaction in MetaMask, it appears as "Pending" or "Queued" for longer than expected. What should I do? 
```

Reply:
Not for now, I was simply making small edits